airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/connectors/upgrade_cdk/pipeline.py

```diff
@@ -0,0 +1,91 @@
+#
+# Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+#
+
+import os
+import re
+
+from pipelines.airbyte_ci.connectors.context import ConnectorContext
+from pipelines.airbyte_ci.connectors.reports import ConnectorReport
+from pipelines.helpers import git
+from pipelines.models.steps import Step, StepResult, StepStatus
+
+
+class SetCDKVersion(Step):
+    title = "Set CDK Version"
+
+    def __init__(
+        self,
+        context: ConnectorContext,
+        new_version: str,
+    ):
+        super().__init__(context)
+        self.new_version = new_version
+
+    async def _run(self) -> StepResult:
+        context: ConnectorContext = self.context
+        og_connector_dir = await context.get_connector_dir()
+        if not "setup.py" in await og_connector_dir.entries():
```

Comment:
Can we return `self.skip(reason="Connector is not a Python connector")` when the connector is a Java connector?